tests/torch/ptq/test_weights_compression.py

```diff
@@ -327,3 +354,48 @@ def test_pack_int4():
     assert packed_w.numel() * 2 == w_int8.numel()
     unpacked_w = unpack_int4(packed_w).reshape(w_int8.shape)
     assert torch.all(unpacked_w == w_int8)
+
+
+class TestPTTemplateWeightCompression(TemplateWeightCompression):
+    @staticmethod
+    def get_matmul_model() -> torch.nn.Module:
+        return MatMulModel(255 * torch.eye(3, dtype=torch.float32))
+
+    @staticmethod
+    def get_sequential_matmul_model() -> torch.nn.Module:
+        return SequentialMatmulModel()
+
+    @staticmethod
+    def to_tensor(t) -> torch.Tensor:
+        return torch.tensor(t)
+
+    @staticmethod
+    def cast_to(x: torch.Tensor, dtype: TensorDataType) -> torch.Tensor:
+        return cast_to(x, dtype)
+
+    @staticmethod
+    def check_weights(model: torch.nn.Module, ref_ids: List[int]) -> None:
+        low_precision_nodes = {f"{i}_weight" for i in ref_ids}
```

Comment:
Please move model names from the testing code to model. 
```python
all_names = model.get_weight_names_in_exec_order()
low_precision_nodes = list(map(lambda i: all_names[i], ref_ids))
for op_name, op in model.nncf.external_op.items():
    for name in low_precision_nodes:
          if name in op_name:
              assert isinstance(op, INT4SymmetricWeightsDecompressor)
```

Reply:
done

Reply:
I can't find where it's done: f"{i}_weight" is still there